src/plugin.js

```diff
@@ -9,16 +9,19 @@ import VREffect from 'three/examples/js/effects/VREffect.js';
 import OrbitOrientationContols from './orbit-orientation-controls.js';
 import * as utils from './utils';
 import CanvasPlayerControls from './canvas-player-controls';
+import OmnitoneController from './omnitone-controller';
```

Comment:
I wonder if we should have a build without spacial audio, as omnitone is quite big. @gkatsev  thoughts?

Reply:
With respect to the size of videojs-vr, omnitone isn't that big, however, I think having it be passed in rather than including it as a dependency might make more sense right now.

Reply:
It sounds good for me. I will make a change based on your suggestions.